web.py

```diff
@@ -139,5 +187,38 @@ def compile(emit, optimize, version, color, syntax):
         else:
             return {"result": split[1].decode()}
 
-os.chdir(sys.path[0])
-run(host='0.0.0.0', port=80, server='cherrypy')
+def main(args):
+    '''
+    -d: Debugging mode: turn on verbose debugging (and RUST_BACKTRACE=1)
+        Change the listening port to 8080 from the default 80.
+    -S: Skip playpen and direcly run rustc and the output binary.
+        To mitigate the risk, accepts only local connections by default.
```

Comment:
Not that I tried it or that I'd recommend doing this, but you can apparently chroot from Debian into a bootstrap image, which might have enough Arch tools to run init.sh: https://wiki.archlinux.org/index.php/Install_from_existing_Linux#From_a_host_running_another_Linux_distribution